pom.xml

```diff
@@ -13,7 +13,7 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <vaadin.version>19.0-SNAPSHOT</vaadin.version>
+        <vaadin.version>20.0-SNAPSHOT</vaadin.version>
```

Comment:
The target branch is `v19` .
Is this PR targets a wrong branch?

Reply:
Yup, my bad. Opened it to v19, although even the pr title said master 😊